exporter/elasticsearchexporter/attribute.go

```diff
@@ -37,3 +38,12 @@ func getFromAttributes(name string, resource, scope, record attrGetter) string {
 	}
 	return str
 }
+
+func getFromAttributesNew(name string, defaultValue string, attributeMaps ...pcommon.Map) string {
```

Comment:
I want to create a separate pull request where I replace the deprecated `getFromAttributes` function with the new one.